terraform/modules/emr/templates/emr/poll_status_table.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+if [[ -n "$1" ]]; then
+  DATASOURCE="$1"
+else DATASOURCE="PDM"
+fi
+
+if [[ -n "$2" ]]; then
+  TIMEOUT="$2"
+else
+  TIMEOUT=3600
```

Comment:
So timeout is in seconds? Would rather it be in minutes to make it easier for a user to use.